plugin-build/src/main/kotlin/io/sentry/android/gradle/tasks/SentryUploadProguardMappingsTask.kt

```diff
@@ -8,12 +8,16 @@ import org.gradle.api.file.RegularFile
 import org.gradle.api.file.RegularFileProperty
 import org.gradle.api.provider.Property
 import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.CacheableTask
 import org.gradle.api.tasks.Exec
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.InputFile
 import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
 
+@CacheableTask
```

Comment:
should this be a separate PR to fix https://github.com/getsentry/sentry-android-gradle-plugin/issues/117 ?

Reply:
oops I actually didn't want to push this. I was checking the Gradle warnings raised by this and forgot to clean my working dir. I'm amending this.